igvm/kvm.py

```diff
@@ -243,14 +243,70 @@ def migrate_live(source, destination, vm, domain):
         min(source.dataset_obj['num_cpu'], destination.dataset_obj['num_cpu']),
     )
 
-    source.run(
-        migrate_cmd.format(
-            domain=domain.name(),
-            destination=destination.fqdn,
-        ),
-        with_sudo=False,
+    migrate_flags = (
+        VIR_MIGRATE_LIVE |  # Do it live
+        VIR_MIGRATE_PERSIST_DEST |  # Define the VM on the new host
+        VIR_MIGRATE_CHANGE_PROTECTION |  # Protect source VM
+        VIR_MIGRATE_NON_SHARED_DISK |  # Copy non-shared storage
+        VIR_MIGRATE_AUTO_CONVERGE |  # Slow down VM if can't migrate memory
```

Comment:
What is the effect here ? How does it "slow down" the VM ?

Reply:
Also this is the exact copy of the old parameters. We might change them to post-copy migration for performance reasons later.